samples/attested_tls/client/enc/CMakeLists.txt

```diff
@@ -65,8 +65,13 @@ elseif (${OE_CRYPTO_LIB} STREQUAL "mbedtls")
     tls_client_enc openenclave::oeenclave openenclave::oecryptombedtls
     openenclave::oelibcxx openenclave::oehostsock openenclave::oehostresolver)
 elseif (${OE_CRYPTO_LIB} STREQUAL "openssl_symcrypt_fips")
-  configure_file(${CMAKE_BINARY_DIR}/SymCrypt/lib/libsymcrypt.so
-                 ${CMAKE_CURRENT_BINARY_DIR}/libsymcrypt.so COPYONLY)
+  add_custom_target(
```

Comment:
A custom target is always out of data and will be re-executed every time.
Can't you instead make this a custom command?

Reply:
updated.